src/components/UserInput.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react"
+
+const UserInput = (props) => {
+  const [enteredInput, setEnteredInput] = useState('')
```

Comment:
What sort of input is this? Generic input, a specific type of input like search, or a specific type of data like postal code?